framework/composer.json

```diff
@@ -71,12 +71,14 @@
         "yiisoft/yii2-composer": "~2.0.4",
         "psr/simple-cache": "~1.0.0",
         "psr/http-message": "~1.0.0",
-        "ezyang/htmlpurifier": "~4.6",
         "cebe/markdown": "~1.0.0 | ~1.1.0",
         "bower-asset/jquery": "3.2.*@stable | 3.1.*@stable | 2.2.*@stable | 2.1.*@stable | 1.11.*@stable | 1.12.*@stable",
         "bower-asset/punycode": "1.3.*",
         "bower-asset/yii2-pjax": "~2.0.1"
     },
+    "suggest": {
+        "ezyang/htmlpurifier": "required at `yii\\helpers\\HtmlPurifier` for 'html' data format support (e.g. `yii\\i18n\\Formatter:asHtml()`)"
```

Comment:
Does Symfony's Console under Composer support/highlight grave accents? If it doesn't, there's no need to use it in the description.